altianalysis/run_difference_with_gpao.py

```diff
@@ -207,7 +236,8 @@ def parse_args():
     args = parse_args()
 
     compute_on_gpao(
-        dtms_lhd=args.dtm_lhd_dir,
+        dtms_lhd=args.primary_dtm_dir,
```

Comment:
c'est plus à jour, je pense que le main ne fonctionne plus dans l'état actuel : 

```suggestion
        primary_dir=args.primary_dtm_dir,
```